Expensify/App
# Add ability to Invite user(s) to a workspace in offline mode

Add ability to invite user in offline mode and display offline info in the bottom of the member page.

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/6676

### Tests



1. Open workspace member page
2. Turn off internet/wifi
3. Verify offline message appear at the bottom of the page
4. Go to invite page and invite some users
5. Verify invited  users emails/name is grayed out
6. Turn on wifi and wait until re-sync icon indicator disappear and please make sure invite API request success
7. Go to HOME url
8. then navigate to that MemberPage
9. Verify the grayed out members displayed  same as other (in black color)
10. Repeat steps from 1-5.
11. Close browser tab 
12. Turn on Wifi
13. Open Home URL and please make sure Invite API request success
14. Verify the grayed out members displayed  same as other (in black color)


### QA Steps



1. Open workspace member page
2. Turn off internet/wifi
3. Verify offline message appear at the bottom of the page
4. Go to invite page and invite some users
5. Verify invited  users emails/name is grayed out
6. Turn on wifi and wait until re-sync icon indicator disappear and please make sure invite API request success
7. Go to HOME url
8. then navigate to that MemberPage
9. Verify the grayed out members displayed  same as other (in black color)
10. Repeat steps from 1-5.
11. Close browser tab 
12. Turn on Wifi
13. Open Home URL and please make sure Invite API request success
14. Verify the grayed out members displayed  same as other (in black color)

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/1313124/152715282-ec41c23f-7049-46fc-ad83-f205011a4674.mp4

#### Mobile Web


https://user-images.githubusercontent.com/1313124/152715913-5c914f30-878f-40f8-bccf-a38ff5b8757c.mp4


#### Desktop


#### iOS


#### Android


https://user-images.githubusercontent.com/1313124/152715865-42631d0d-b419-42b5-8945-3be83c1df835.mp4
